tests/e2e/default/tasks/tasks.wdio-spec.js

```diff
@@ -41,8 +130,68 @@ describe('Tasks', () => {
     await browser.refresh();
   });
 
-  afterEach(async () => {
-    await utils.revertSettings(true);
+  it('should load multiple pages of tasks on infinite scrolling', async () => {
+    const settings = await compileTasks('tasks-multiple-config.js');
+    await utils.updateSettings(settings, { ignoreReload: 'api', sync: true });
+
+    await commonPage.goToTasks();
+    const list = await tasksPage.getTasks();
+    const infos = await tasksPage.getTasksListInfos(list);
+    expect(infos).to.have.length(201);
+    for (let i = 0; i < (infos.length/3); i++) {
+      expect(infos).to.include.deep.members([
+        {
+          contactName: 'Mary Smith',
+          formTitle: `person_create_${i + 1}`,
+          lineage: '',
+          dueDateText: 'Due today',
+          overdue: true
+        },
+        {
+          contactName: 'patient1',
+          formTitle: `person_create_${i + 1}`,
+          lineage: 'clinic2',
+          dueDateText: 'Due today',
+          overdue: true
+        },
+        {
+          contactName: 'patient2',
+          formTitle: `person_create_${i + 1}`,
+          lineage: '',
+          dueDateText: 'Due today',
+          overdue: true
+        },
+      ]);
+    }
+    await tasksPage.scrollToLastTaskItem();
+    const loadingStatusSelector = await $('#tasks-list p.loading-status');
+    const elementText = await loadingStatusSelector.getText();
+    expect(elementText).to.contain('No more tasks');
+  });
+
+  it('should remove task from list when CHW completes a task successfully', async () => {
+    const settings = await compileTasks('tasks-breadcrumbs-config.js');
+    await utils.updateSettings(settings, { ignoreReload: 'api', sync: true });
+    
+    await commonPage.goToTasks();
+    let list = await tasksPage.getTasks();
+    expect(list).to.have.length(3);
+    await tasksPage.submitTask(1);
+    await commonPage.sync(true);
+    list = await tasksPage.getTasks();
+    expect(list).to.have.length(2);
+  });
+
+  // WIP
+  it('should add a task when CHW completes a task successfully, and that task creates another task', async () => {
+    const settings = await compileTasks('tasks-breadcrumbs-config.js');
+    await utils.updateSettings(settings, { ignoreReload: 'api', sync: true });
+    
+    await commonPage.goToTasks();
+    const list = await tasksPage.getTasks();
+    expect(list).to.have.length(2);
+    await tasksPage.submitTask(1);
+    // validate new task is created = taskList length should still be 2
```

Comment:
Is this still in progress?

Reply:
yes this is still being worked but may yank it off since the logic to validate this is not forthcoming

Reply:
ok, but before we merge this PR to `master` we will need to complete this test or remove it

Reply:
sure that's noted